### ISSUE TITLE
The `decimalCharacterAlternative` option does not ignore the comma when it's set to `none`

### ISSUE THREAD

**Issue by AlexandreBonneau** [autoNumeric/autoNumeric#432]
There is a bug when using the `anElement.options.decimalCharacterAlternative()` method in conjunction with the following options:
```js
decimalCharacter             : AutoNumeric.options.decimalCharacter.comma,
decimalCharacterAlternative  : AutoNumeric.options.decimalCharacterAlternative.dot,
```

If you initialize an element:
```js
new AutoNumeric('#optionUpdate', 4444.66)
    .french()
    .options.decimalCharacterAlternative(AutoNumeric.options.decimalCharacterAlternative.none);
```
then this should means the way to add a decimal character should only be using the *comma* (`settings.decimalCharacter`) since I removed any `decimalCharacterAlternative`, but even with this, I can still use the `dot` to add one.

Since the user specifically asked for no alternate decimal character `decimalCharacterAlternative`, we should not allow recognize the `dot` as one.

The same problem arise when using:
```js
new AutoNumeric('#optionUpdate3', 8888.00).northAmerican()
```
where the element accepts either the `','` or the `'.'` as the decimal character, while `decimalCharacterAlternative` is set to `null` by default.

**Comment by karlingen** [comment]
I'm having similar issue on `4.0.3`. 

```
digitGroupSeparator: '.',
    decimalCharacter: ',',
```
It adds a decimal when dot is pressed, which makes no sense IMO. 
It should either ignore the dot or add a dot and not a decimal character.

**Comment by AlexandreBonneau** [comment]
This is fixed in `v4.1.0-beta.27`
